contentful_management/resource.py

```diff
@@ -274,9 +274,7 @@ def create_attributes(klass, attributes, previous_object=None):
 
         result = super(MetadataResource, klass).create_attributes(attributes, previous_object)
 
-        if '_metadata' not in attributes:
-            result['metadata'] = {}
-        else:
+        if '_metadata' in attributes:
             result['metadata'] = attributes['_metadata']
```

Comment:
for backward compatibility does it need to be set to empty object if not present still? this will be null/None now won't it?

Reply:
also couldn't this just be expressed as ```result['metadata'] = attributes['_metadata'] || {}``` to simplify this?

Reply:
We don't want to set the `metadata` key if no metadata are present in attributes otherwise API will raise error